manifests/python.yml

```
@@ -81,19 +81,19 @@ tests/:
         test_insecure_cookie.py:
           TestInsecureCookie:
             '*': v1.19.0
-            fastapi: missing_feature
+            fastapi: v2.16.0dev
```

Just to have clean semver declaration

```suggestion
            fastapi: v2.16.0-dev
```

---

with that, I can open a new PR because we have many values wrong